compiler/src/dotty/tools/dotc/reporting/ErrorMessageID.scala

```diff
@@ -176,7 +176,8 @@ enum ErrorMessageID extends java.lang.Enum[ErrorMessageID]:
     MatchableWarningID,
     CannotExtendFunctionID,
     LossyWideningConstantConversionID,
-    ImplicitSearchTooLargeID
+    ImplicitSearchTooLargeID,
+    TargetNameOnTopLevelClassID
```

Comment:
Does anyone else feel the need for an end marker, and also trailing comma before end marker?